ui/pages/settings/experimental-tab/experimental-tab.component.tsx

```diff
@@ -0,0 +1,314 @@
+    });
+  }
+
+  renderNotificationsToggle() {
+    const { t } = this.context;
+    const { featureNotificationsEnabled, setFeatureNotificationsEnabled } =
+      this.props;
+
+    return this.renderToggleSection({
+      title: t('notificationsFeatureToggle'),
+      description: t('notificationsFeatureToggleDescription'),
+      toggleValue: featureNotificationsEnabled,
+      toggleCallback: (value) => setFeatureNotificationsEnabled(!value),
+      toggleDataTestId: 'toggle-notifications',
+      toggleOffLabel: t('off'),
+      toggleOnLabel: t('on'),
+    });
+  }
+
+  ///: BEGIN:ONLY_INCLUDE_IF(build-flask,build-beta)
+  // We're only setting the code fences here since
+  // we should remove it for the feature release
+  renderBitcoinSupport() {
+    const { t, trackEvent } = this.context;
+    const { bitcoinSupportEnabled, setBitcoinSupportEnabled } = this.props;
+
+    return (
+      <>
+        <Text
+          variant={TextVariant.headingSm}
+          as="h4"
+          color={TextColor.textAlternative}
+          marginBottom={2}
+          fontWeight={FontWeight.Bold}
+        >
+          {t('experimentalBitcoinSectionTitle')}
+        </Text>
+        {this.renderToggleSection({
+          title: t('experimentalBitcoinToggleTitle'),
+          description: t('experimentalBitcoinToggleDescription', [
+            <a
+              key="btc-account-feedback-form__link-text"
+              href=""
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {t('form')}
+            </a>,
+          ]),
+          toggleValue: bitcoinSupportEnabled,
+          toggleCallback: (value) => {
+            trackEvent({
+              event: MetaMetricsEventName.BtcExperimentalToggled,
+              category: MetaMetricsEventCategory.Settings,
+              properties: {
+                enabled: !value,
+              },
+            });
+            setBitcoinSupportEnabled(!value);
+          },
+          toggleDataTestId: 'bitcoin-accounts-toggle',
+          toggleOffLabel: t('off'),
+          toggleOnLabel: t('on'),
+        })}
+      </>
+    );
+  }
+  ///: END:ONLY_INCLUDE_IF
+
+  render() {
+    return (
+      <div className="settings-page__body">
+        {this.renderTogglePetnames()}
+        {this.renderToggleRedesignedConfirmations()}
+        {process.env.NOTIFICATIONS ? this.renderNotificationsToggle() : null}
+        {this.renderToggleRequestQueue()}
+        {/* Section: Account Management Snaps */}
+        {
+          ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
+          this.renderKeyringSnapsToggle()
+          ///: END:ONLY_INCLUDE_IF
+        }
+        {
+          ///: BEGIN:ONLY_INCLUDE_IF(build-flask,build-beta)
+          // We're only setting the code fences here since
+          // we should remove it for the feature release
+
+          /* Section: Bitcoin Accounts */
+          <>{this.renderBitcoinSupport()}</>
```

Comment:
Cannot we just use this instead?
```suggestion
          this.renderBitcoinSupport()
```

Reply:
[736dc39](https://github.com/MetaMask/metamask-extension/pull/25672/commits/736dc39df7c2c3b8b3de78d6fa662fb18776e58d)